editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -1876,8 +1874,8 @@ SpriteFramesEditor::SpriteFramesEditor() {
 	frame_list->set_max_text_lines(2);
 	SET_DRAG_FORWARDING_GCD(frame_list, SpriteFramesEditor);
 	frame_list->connect("gui_input", callable_mp(this, &SpriteFramesEditor::_frame_list_gui_input));
-	frame_list->connect("item_selected", callable_mp(this, &SpriteFramesEditor::_frame_list_item_selected));
-
+	// HACK: The item_selected signal is emitted before the Frame Duration spinbox loses focus and applies the change.
```

Comment:
Oh no a hack.

Reply:
I think we can avoid this if we track which frame was selected when focus was gain and update that frame on focus loss, not the one currently selected. See https://github.com/godotengine/godot/pull/79692#discussion_r1341426434.